### PR TITLE
Removes references to BitBucket

### PR DESCRIPTION
**Pull request by GlynnPhillips**
BitBucket is no longer used for Next repositories so removing references
to it in the documentation.

As Jenkins was only used for BitBucket repositories the page containing
Jenkins setup information has also been removed.

### CODE REVIEW

**Comment by commuterjoy** [comment]
+1

**Comment by tom-parker** [comment]
:100:

File: docs/developer-guide/new-starter.md [hunk 1]
```diff
@@ -19,10 +19,9 @@ permalink: /docs/developer-guide/new-starter/
 - Add yourself to our [bug rota](https://docs.google.com/spreadsheets/d/1mbJQYJOgXAH2KfgKUM1Vgxq8FUIrahumb39wzsgStu0/edit#gid=0) (check whether this is still running)
 - IT Services should have enabled admin rights when they set up your machine, but you should [request admin access to your FT laptop via Salesforce](https://financialtimes.my.salesforce.com/home/home.jsp) to ensure it doesn't get revoked.
 - Repositories
-  * Public repositories are on [GitHub](https://github.com/Financial-Times/), private on [BitBucket](http://git.svc.ft.com/projects/NEXT).  Repositories should be public if at all possible.  Do not create private repositories on GitHub.
+  * Repositories are on [GitHub](https://github.com/Financial-Times/) and should be public if at all possible.
```
Comment (adambraimbridge) [inline docs/developer-guide/new-starter.md]: Is it okay to create private github repos now?

**Comment by adambraimbridge** [comment]
:+1:

File: docs/developer-guide/new-starter.md (hunk 1 above)
Comment (matthew-andrews) [inline docs/developer-guide/new-starter.md]: yes

**Pull request merged**